LobbyManager.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TICTACTOE_SERVER_LOBBYMANAGER_H
+#define TICTACTOE_SERVER_LOBBYMANAGER_H
+
+#include "SearchManager.h"
+#include <unordered_map>
+#include <random>
+
+class LobbyManager {
+public:
+	struct Lobby {
+		SearchManager::PlayerInfo player1;
+		SearchManager::PlayerInfo player2;
+		bool player1_active;
+		bool player2_active;
```

Comment:
These two fields shouldn't be visible outside this class. Refactor it so that they aren't returned from a public method

Reply:
This isn't resolved. Fix it.